MidiFile/MidiEvents/SysexEvents/MidiSysexEvent.h

```diff
@@ -52,7 +52,7 @@ enum EMidiManufacturer : uint16_t
     EXTENSION           = 0xFFFFu
 };
 
-std::string get_manufacturer_string(EMidiManufacturer manufacturer)
+inline std::string get_manufacturer_string(EMidiManufacturer manufacturer)

```

Comment:
Got multiply defined symbol errors when including MidiFile.h in more than 1 .cpp file, this fixes it.